FogCreek/Glitch-Community
# Update Button to enable passing HTML attributes

## Links
* [copper-origami.glitch.me](https://copper-origami.glitch.me)
* [storybook](https://copper-origami.glitch.me/storybook/index.html?path=/story/button--with-aria-label)
* [manuscript case](https://glitch.manuscript.com/f/cases/3328205#BugEvent.21311456)

## Changes
This makes it possible to pass HTML attributes like `aria-label` into the Button component

## How To Test:
See the [example in storybook](https://copper-origami.glitch.me/storybook/index.html?path=/story/button--with-aria-label) or inspect the pagination buttons on the recent projects section of a [team page](https://copper-origami.glitch.me/@glitch).

## Feedback I'm looking for:
Are we ok with passing in HTML attributes via `props` or do we prefer to specifically define a `label` or `ariaLabel` prop?
